crates/frontend/src/pages/function/function_list.rs

```diff
@@ -24,7 +27,15 @@ struct CombinedResource {
 pub fn function_list() -> impl IntoView {
     let tenant_rws = use_context::<RwSignal<Tenant>>().unwrap();
     let org_rws = use_context::<RwSignal<OrganisationId>>().unwrap();
-    let (filters, set_filters) = create_signal(PaginationParams::default());
+    let (filters, set_filters) = use_signal_from_query(move |query_string| {
```

Comment:
```suggestion
    let (pagination_params_rs, pagination_params_ws) = use_signal_from_query(move |query_string| {
```
This but everywhere that (filters, set_filters) are used

Reply:
I too wanted to do that, but didn't want to increase the change log, but sure, adding the change as well